tests/testthat/test-facets.R

```diff
@@ -47,3 +47,49 @@ test_that("3 facets becomes 3 panels", {
   u <- unique(trace.axes.df)
   expect_identical(nrow(u), 3L)
 })
+
+# expect a certain number of _unique_ [x/y] axes
```

Comment:
Is this renaming just to reflect that facets should behave exactly like they do in ggplot2? Just fyi, we could still implement `space = "free"`